spec/models/lockstep/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Lockstep::User' do
+
+  context 'when calling User Invite to invite a user' do
+    it 'should return true as the email is not present already' do
+      VCR.use_cassette("models/lockstep/user/exist") do
+        response = Lockstep::User.invite(["abcd@gmail.com"])
+        expect(response.first['success']).to eq(true)
```

Comment:
How is this passing? the code seems to have changed.

Reply:
Lets make these test cases more thorough. Lets check all all the attributes to ensure its correct

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/82f68ab70e0de00e786a23cee048057f11b2e80c)